examples/Coffee_Machine_FSM/coffee.py

```diff
@@ -0,0 +1,201 @@
+from enum import Enum
+from typing import Union, List
+
+
+
+class StatusCode(Enum):
+    OK = 1
+    ERROR = 0
+    COFFEE_PRODUCED = 2
+
+
+
+class State:
+    def __init__(self, name: str, coffee_machine: "CoffeeMachine") -> None:
+        self.name: str = name
+        self.coffee_machine: CoffeeMachine = coffee_machine
+
+    def handle_input(self, input_alphabet: Union[str, List[str]]) -> int:
```

Comment:
Make these methods abstract methods such that you do not have to throw the error here.